packages/excalidraw/tests/cropElement.test.tsx

```diff
@@ -158,7 +150,9 @@ describe("Cropping and other features", async () => {
     ]);
     Keyboard.keyDown(KEYS.ESCAPE);
     const duplicatedImage = duplicateElement(null, new Map(), image, {});
-    h.app.scene.insertElement(duplicatedImage);
+    act(() => {
+      h.app.scene.insertElement(duplicatedImage);
+    });
```

Comment:
@ryan-di make sure that all state mutations in tests are wrapped in `act()`, or use helpers that are doing it themselves, such as the `API.updateElement()` instead of `mutateElement()` (see below). This makes it more deterministic the way I understand it. If nothing else, it removes the pesky warnings in console for cleaner output.

Reply:
Understood!